about.html

```diff
@@ -85,16 +78,16 @@ <h2 class="sponsorTitle">sponsors</h2>
   </div>
 </section>
 
-<section id="darkLicense">
+<section id="license1">
   <img class="licence-logo" src="images/creative-commons-license-logo.jpg" alt="licence-logo"/>
   <div>
     <p class="licence-text">2015 Creative Commons <i class="fa fa-creative-commons" aria-hidden="true"></i></p>
-    <p class="licence-text">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a></p>
+    <p class="licence-text">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International license</a></p>
   </div>
 </section>
 <section class="mobile-license">
   <img class="licence-logo" src="images/creative-commons-license-logo.jpg" alt="licence-logo"/>
-  <p class="licence-text" id="licence-mobile">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a></p>
+  <p class="licence-text" id="licence-mobile">This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International license</a></p>
 </section>
```

Comment:
- [ ] The about page does not have a partner's section but it has only a dark footer section. It would be best if you remove the partner's section from your page so as to follow the design requirements.
- Your design:
![Capture2](https://user-images.githubusercontent.com/85113567/153715257-10aab00f-2649-43d7-bd60-eb5e3cd4c583.PNG)
- Required design:
![Capture90](https://user-images.githubusercontent.com/85113567/153715276-f017497a-6516-490b-8510-b9c5e7147404.PNG)

Reply:
This issue has been resolved